meshtastic/mesh_interface.py

```diff
@@ -246,6 +279,29 @@ def getTimeAgo(ts) -> Optional[str]:
                 return None  # not handling a timestamp from the future
             return _timeago(delta_secs)
 
+        def getNestedValue(node_dict: Dict[str, Any], key_path: str) -> Any:
+            if key_path.index(".") < 0:
+                logging.debug("getNestedValue was called without a nested path.")
+                return None
+            keys = key_path.split(".")
+            value = node_dict
+            for key in keys:
+                if isinstance(value, dict):
+                    value = value.get(key)
+                else:
+                    return None
+            return value
+
+        if showFields is None or showFields.count == 0:
```

Comment:
I'm pretty sure that `count` is a method on lists, not a property, so this wouldn't ever be true. I'm guessing you mean `len(showFields) == 0` here.